### PR TITLE
hide upgrade feature from help command and docs

### PR DESCRIPTION
**Pull request by sinhaashish**
Since the upgarade is meant for release 2.0+ versions, hiding the upgrade feature from help and docs for now.

Signed-off-by: sinhaashish <ashi.sinha.87@gmail.com>

### CODE REVIEW

**Comment by sinhaashish** [comment]
bors try

**Review by niladrih — COMMENTED** [review]
Maybe we should also disable the code paths for upgrade, like so...
https://github.com/openebs/mayastor-extensions/blob/9153fd123b77fa0a68a4f2d4fc420caf57f77766/k8s/plugin/src/main.rs#L118
```rust
                GetResourcesK8s::UpgradeStatus => {
                     // No-op  because upgrade to 2.0 is not supported.                 
/*                  UpgradeResources::get(
                        cli_args.upgrade_operator_endpoint,
                        &cli_args.namespace,
                        cli_args.kube_config_path,
                        cli_args.timeout,
                    )
                    .await
*/
                }
```

File: k8s/plugin/src/resources/mod.rs [hunk 1]
```diff
@@ -36,11 +36,12 @@ pub enum Operations {
     /// `Dump` resources.
     Dump(DumpArgs),
     /// `Install` upgrade operator.
-    #[clap(subcommand)]
+    #[clap(subcommand, hide = true)]
```
Comment (tiagolobocastro) [inline k8s/plugin/src/resources/mod.rs:39]: Should we remove the clap macro completely so it can't even be called?

File: k8s/plugin/src/main.rs [hunk 2]
```diff
@@ -53,7 +49,7 @@ struct CliArgs {
 
     /// Endpoint of upgrade operator service, if left empty then it will try to parse endpoints
     /// from upgrade operator service(K8s service resource).
-    #[clap(global = true, short, long)]
+    #[clap(global = true, short, long, hide = true)]
```
Comment (tiagolobocastro) [inline k8s/plugin/src/main.rs:52]: can also delete `upgrade_operator_endpoint` now I guess

File: scripts/release.sh [hunk 3]
```diff
@@ -65,8 +65,8 @@ HASH=`get_hash`
 BRANCH=`git rev-parse --abbrev-ref HEAD`
 BRANCH=${BRANCH////-}
 IMAGES=
-DEFAULT_IMAGES="exporters.metrics.pool operators.upgrade obs.callhome"
-IMAGES_THAT_REQUIRE_HELM_CHART=("operators.upgrade")
+DEFAULT_IMAGES="exporters.metrics.pool obs.callhome"
+IMAGES_THAT_REQUIRE_HELM_CHART=
```
Comment (tiagolobocastro) [inline scripts/release.sh:69]:
```suggestion
IMAGES_THAT_REQUIRE_HELM_CHART=()
```

**Review by tiagolobocastro — APPROVED** [review]

**Review by niladrih — APPROVED** [review]

**Comment by sinhaashish** [comment]
bors merge

**Comment by sinhaashish** [comment]
bors merge

**Pull request merged**